apps/server/src/modules/authentication/strategy/local.strategy.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { PassportModule } from '@nestjs/passport';
+import { Test, TestingModule } from '@nestjs/testing';
+
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { UnauthorizedException } from '@nestjs/common';
+import bcrypt from 'bcryptjs';
+import { UserRepo } from '@shared/repo';
+import { AccountDto } from '@src/modules/account/services/dto';
+import { Role, RoleName, School, User } from '@shared/domain';
+import { Collection } from '@mikro-orm/core';
+import { LocalStrategy } from './local.strategy';
+import { AuthenticationService } from '../services/authentication.service';
+
+const mockPassword = 'mockPassword123&';
+const mockPasswordHash = bcrypt.hashSync(mockPassword);
+
+const studentRole: Role = { id: 'mockRoleId', name: RoleName.STUDENT } as Role;
+const mockUser: User = {
+	id: 'mockUserId',
+	roles: new Collection<Role>(null, [studentRole]),
+	school: { id: 'mockSchoolId' } as School,
+} as User;
+const mockAccount = {
```

Comment:
Same like the others, use factorys, when senario describes and setup methodes.

Reply:
Done.